docs/lcms/unified.yaml

```diff
@@ -56,9 +56,10 @@ fields:
   name: assay_category
 - constraints:
     enum:
-    - LC-MS (metabolomics)
-    - LC-MS/MS (label-free proteomics)
-    - MS (shotgun lipidomics)
+    - Label-free LC-MS
+    - Labeled LC-MS
+    - Label-free LC-MS/MS
+    - Labeled LC-MS/MS
```

Comment:
Confirned with Kevin Contrepois, Daniele Gutierrex & Lihua Jiang. These 3 names cover all mass spec assays expected:
Labeled LC-MS/MS
Label-free LC-MS/MS
Label-free MS/MS

Reply:
Is this a typo?
> Label-free MS/MS

Reply:
Not a typo. Made PR in search api.